### PR TITLE
Add and remove accounts in the wallet

### LINKED ISSUE #63: Add support to multiple accounts in the same wallet (NEP-6)
**Summary or problem description**
Wallets can have multiple accounts. This issue is for including new accounts to existing wallets and creating new wallets with multiple accounts

**Additional contextual information**
- A NEP-6 wallet is a collection of accounts, meaning that it can contain one or multiple NEP-2 accounts.
  - The wallet should be already implemented in #59. This issue is to include accounts to the wallet

[AddAccount(acc: Account)
RemoveAccount(address: str)](https://github.com/nspcc-dev/neo-go/blob/b0e8e37b99497ada3c5edd0d7f115f60aeba0aab/pkg/wallet/wallet.go#L103-L118)

### PR DESCRIPTION
**Pull request by meevee98**
implemented the methods to include and remove accounts

closes #63

### CODE REVIEW

File: neo3/wallet/wallet.py [hunk 1]
```diff
@@ -75,10 +76,73 @@ def __init__(self,
...
+        self.account_add(account, is_default)
+        return account
+
+    def account_add(self, account: Account, is_default=False) -> bool:
+        # true if ok, false if duplicate (any other possible reasons? otherwise we need to throw exceptions)
+        if account in self.accounts:
+            return False
+
+        if account.label is not None and self._get_account_by_label(account.label) is not None:
+            raise ValueError("Label is already used by an account '{0}'", account.label)
```
Comment (ixje) [inline neo3/wallet/wallet.py:109]: Please use f-strings, which is used in the rest of mamba.

File: neo3/wallet/wallet.py [hunk 2]
```diff
@@ -75,10 +76,73 @@ def __init__(self,
...
+        # return success or not
+        account = self._get_account_by_label(label)
+
+        if account is None:
+            # account with that label was not found
+            return False
+
+        return self.account_delete(account)
+
+    def _get_account_by_label(self, label: str) -> Optional[Account]:
```
Comment (ixje) [inline neo3/wallet/wallet.py:141]: I think this method would be useful if its public as `account_get_by_label`. We can later on add more `account_get_*` helpers

**Review by ixje — CHANGES_REQUESTED** [review]
Can you also add doc strings?

**Comment by ixje** [comment]
Can you please solve the conflict? I've asked Ricardo if I can be added to the mamba repo at the simplitech org, because I can't add commits and move this forward due to this https://github.com/isaacs/github/issues/1681#issue-508922256

**Comment by meevee98** [comment]
Done

**Review by ixje — APPROVED** [review]
THanks! 🥇

**Pull request merged**